out_camera_data.xml

```diff
@@ -0,0 +1,623 @@
+<?xml version="1.0"?>
```

Comment:
No, probably not, the .yml file is no longer necessary (I will remove it), where would it best fit?